src/test/fuzz/integer.cpp

```diff
@@ -279,4 +279,8 @@ void test_one_input(const std::vector<uint8_t>& buffer)
         catch (const std::ios_base::failure&) {
         }
     }
+
+    {
+        CScriptNum::serialize(i64);
+    }
```

Comment:
Also adjust line 139 of this file?

Reply:
Ow, right, I missed this one when I grepped for the runtime error.

Hence removed the `CScriptNum::serialize(i64);` since line 139 already tests it.